website/frontend/src/components/blocks/Header/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { VFC, useState } from 'react';
+import close from 'assets/images/close.svg';
+import refresh from 'assets/images/refresh2.svg';
+import selected from 'assets/images/radio-selected.svg';
+import deselected from 'assets/images/radio-deselected.svg';
+import copy from 'assets/images/copy.svg';
+import { copyToClipboard, isNodeAddressValid } from 'helpers';
+import { Trash2 } from 'react-feather';
+import { nodeApi } from 'api/initApi';
+import { useAlert } from 'react-alert';
+import { useDispatch } from 'react-redux';
+import { resetApiReady } from 'store/actions/actions';
+import { useHistory, useLocation } from 'react-router-dom';
+import { NODE_ADRESS_URL_PARAM } from 'consts';
+import * as init from './init';
+import './Sidebar.scss';
+
+type Props = {
+  closeSidebar: () => void;
+};
+
+const Sidebar: VFC<Props> = ({ closeSidebar }) => {
+  const dispatch = useDispatch();
+  const alert = useAlert();
+  const location = useLocation();
+  const history = useHistory();
+
+  const [nodes, setNodes] = useState(localStorage.nodes ? JSON.parse(localStorage.nodes) : init.nodes);
+
+  const allNodes = [...nodes[0].nodes, ...nodes[1].nodes];
+  const isNodeExist = allNodes.findIndex((node) => node.address === nodeApi.address) > -1;
+
+  const [selectedNode, setSelectedNode] = useState(isNodeExist ? nodeApi.address : '');
+  const [newNode, setNewNode] = useState(isNodeExist ? '' : nodeApi.address);
+
+  const handleAddNode = () => {
+    const nodeToAdd = {
+      id: nodes[1].nodes.length + 2,
+      custom: true,
+      isChoose: false,
+      address: newNode,
+    };
+
+    setNodes((elems: any) =>
```

Comment:
could you implement this one without any?

Reply:
It's a standalone task.